.travis.yml

```diff
@@ -10,6 +10,7 @@ env:
   - TRIGGER_REPO_BRANCH: "master"
 
 before_install:
+  - nvm install 4
   - gem install jekyll -v 3.1.6
   - if [[ `npm -v` != 3* ]]; then npm i -g npm@3; fi
   - npm install -g bower grunt-cli
```

Comment:
Can you explain what's happening regarding jekyll and npm? Not sure what versions are being installed under nvm 4?